scapy/arch/pcapdnet.py

```diff
@@ -149,7 +149,7 @@ def next(self):
           c = pcap_next_ex(self.pcap, byref(self.header), byref(self.pkt_data))
           if not c > 0:
               return
-          ts = self.header.contents.ts.tv_sec
+          ts = float(self.header.contents.ts.tv_sec + decimal.Decimal("."+str(self.header.contents.ts.tv_usec)))
```

Comment:
This computation looks wrong. I think that `tv_usec` must be converted to seconds before adding it to `tv_sec`.

Reply:
I'm not sure to understand what you want to do with `decimal`.

Wouldn't this work?
```Python
          ts = self.header.contents.ts.tv_sec + float(self.header.contents.ts.tv_usec) / 1000000
```

Reply:
Oh right. I thought it was safer to use a such module, but it does seems heavier